charter.md

```diff
@@ -0,0 +1,55 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+## Governance
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition. Decisions can be made both in the repository and in meetings, as long as consensus can be established.
+
+The chair of the team is responsible for organizing meetings, driving discussions to a conclusion, and recording decisions. The team appoints an individual as the chair based on consensus at a minimum once per calendar year. There's no term limit; the same individual can be reappointed as chair. If the team is unable to meet due to absence of a chair, the Core Team may appoint one directly.
```

Comment:
There's a couple of things about the chair... Its very vague.. i guess on purpose, but like.. Can we say how long a term is? Or that the team appoints for a term they specify, up to 1 year? I was thinking about co-chairs and interim chairs which are often helpful in working groups for very practical reasons and wondering if/how something like that would work.. could we say X is temporarily on leave, Y will serve as acting chair until they're back?

Reply:
I assume the "at minimum once per calendar year" means that you have to re-decide on the chair each year (i.e. the maximum duration of an appointment is a year) but you can do it more often than that (i.e. there could be a "temporary" chair under the process). I agree the outcome is somewhat implicit though.

Reply:
My interpretation agrees with James’.

Reply:
Updated the text to clarify the yearly term for the chair